### DISCUSSION TITLE
How can I change the zoom easing/animation?

### DISCUSSION THREAD

**Discussion by cdslxc** [maplibre/maplibre-gl-js#3236] [Q&A]
How can I change the zoom easing/animation?

There is a recipe for change camera animation how about zoom animation?

**Comment by HarelM** [comment]
You can specify an easing function when calling methods like `easeTo` and `flyTo`.
Check out the API documentation and the examples here:
https://maplibre.org/maplibre-gl-js/docs/

**Reply by cdslxc** [reply]
I'm looking for changing easings on scrollZoom/boxZoom. How can I do that?

**Reply by HarelM** [reply]
I don't think you can.
Feel free to reverse engineer the following code:
https://github.com/maplibre/maplibre-gl-js/blob/16aa454a8ea287f66f8a1f1aad5f1a77ee57c1cd/src/ui/handler/scroll_zoom.ts#L250

**Reply by cdslxc** [reply]
@HarelM okay, but does this function also run on touch scroll (eq. Macbook trackpad)?

**Reply by HarelM** [reply]
I think so.
